core/ledger/util/couchdb/couchdbutil_test.go

```diff
@@ -1,184 +0,0 @@
-/*
```

Comment:
Did you delete this test file by mistake?

Reply:
Fixed